website/plugins/gatsby-transformer-react-docgen-typescript/package.json

```diff
@@ -0,0 +1,51 @@
+{
```

Comment:
I would skim this package to the minimum since you are not publishing or anything, I would only leave the name to avoid confusions